README.md

```diff
@@ -1,8 +0,0 @@
-# Workflows
```

Comment:
If you prefer to keep the list in a single place, and don't repeat it, I agree with you, that's perfect.

On the other hand, leave at least the description, and possibly also a link to the docs (directly link the website, like https://qibogang.github.io/workflows/) in the README: the user that arrives to the repo before the website should get at least a description, and a pointer as well.